packages/frontend/components/composed/journal/CurrentIssue/CurrentIssue.styles.ts

```diff
@@ -59,6 +63,8 @@ export const TitleBlock = styled.div`
   }
 `;
 
+export const Subtitle = styled.div``;
```

Comment:
nit: Same for Subtitle - if the tag doesn't have any styles, let's not define them here.